.github/workflows/devcontainer-feature-release.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Release dev container feature"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  call-workflow-test:
+    uses: ./.github/workflows/devcontainer-feature-test.yaml
+
+  
+  deploy:
+    needs: 
+      - call-workflow-test
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Publish Features"
+        uses: devcontainers/action@v1
+        with:
+          publish-features: "true"
+          base-path-to-features: "./ext/devcontainer/src"
+          generate-docs: "false"
```

Comment:
Do you know how versioning works for this?  

/cc @willdavsmith

Reply:
Besides using semantic versioning I did not see any consistent management of versions (yet). As the versioning of the feature  decoupled from the `rad` CLI versioning I think we can stick to changing versions in accordance to changing code that is specific to the feature *per se*

Reply:
Let me suggest then that we don't try to solve versioning and publishing of the devcontainer as part of your PR. I'll resolve all of the comments I left that are related to that.

The maintainers need to decide how versioning/publishing will work because we'll be handling it as part of each release (or some other cadence). I don't want to block your PR while we figure this out.